src/worker/AnalysisWorkerWrapper.js

```diff
@@ -39,38 +36,41 @@ class AnalysisWorkerWrapper {
 	 *
 	 * See: https://developer.mozilla.org/en-US/docs/Web/API/Worker/onmessage
 	 *
-	 * @param {MessageEvent} arguments              The post message event.
-	 * @param {Object}       arguments.data         The data object.
-	 * @param {string}       arguments.data.type    The action type.
-	 * @param {string}       arguments.data.payload The payload of the action.
+	 * @param {MessageEvent} event              The post message event.
+	 * @param {Object}       event.data         The data object.
+	 * @param {string}       event.data.type    The action type.
+	 * @param {number}       event.data.id      The request id.
+	 * @param {string}       event.data.payload The payload of the action.
 	 *
 	 * @returns {void}
 	 */
-	handleMessage( { data: { type, payload } } ) {
-		let id;
-		console.log( "wrapper", type, payload );
+	handleMessage( { data: { type, id, payload } } ) {
+		let response, request;
+		console.log( "wrapper", type, id, payload );
+
 		switch( type ) {
 			case "initialize:done":
```

Comment:
`initialize:done` and `analyze:done` seem to have the same content. Any reason these are two different cases? Is is it because we expect more differing functionality to be implemented very soon?